<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PaxoTechWin7X64" name="FFScript" tests="9" failures="3" timestamp="28 Sep 2015 18:23:06 GMT" time="284.53" errors="0">
  <testcase name="SmSAlart" time="8.002" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality"/>
  <testcase name="boundaryInfo" time="17.629" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality"/>
  <testcase name="emtyInfologin" time="16.039" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality">
    <failure type="org.junit.ComparisonFailure" message="expected:&amp;lt;[It appears that either the email address or password entered is incorrect. Please re-enter and submit the information.]&amp;gt; but was:&amp;lt;[[It appears that either the email address or password entered is incorrect. Please re-enter and submit the information.]]&amp;gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<[It appears that either the email address or password entered is incorrect. Please re-enter and submit the information.]> but was:<[[It appears that either the email address or password entered is incorrect. Please re-enter and submit the information.]]]]><![CDATA[>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.paxotech.qa.lord.LoginBasePage.ivalidEamil(LoginBasePage.java:42)
	at com.lordandTaylor.testsripts.LoginBaseFanctionality.emtyInfologin(LoginBaseFanctionality.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:391)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- emtyInfologin -->
  <testcase name="invalidEmailAddforReg" time="16.617" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality"/>
  <testcase name="invalidRegistration" time="17.138" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality"/>
  <testcase name="invlaidUerLogin" time="16.277" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality">
    <failure type="org.junit.ComparisonFailure" message="expected:&amp;lt;[It appears that either the email address or password entered is incorrect. Please re-enter and submit the information.]&amp;gt; but was:&amp;lt;[[It appears that either the email address or password entered is incorrect. Please re-enter and submit the information.]]&amp;gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<[It appears that either the email address or password entered is incorrect. Please re-enter and submit the information.]> but was:<[[It appears that either the email address or password entered is incorrect. Please re-enter and submit the information.]]]]><![CDATA[>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.paxotech.qa.lord.LoginBasePage.ivalidEamil(LoginBasePage.java:42)
	at com.lordandTaylor.testsripts.LoginBaseFanctionality.invlaidUerLogin(LoginBaseFanctionality.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:391)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- invlaidUerLogin -->
  <testcase name="newMemberRegistration" time="20.029" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality"/>
  <testcase name="verifyhomepagefaunctionality" time="10.358" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality">
    <failure type="java.lang.AssertionError" message="expected [] but found [Lord and Taylor: Designer Clothing, Shoes, Handbags, Accessories &amp;amp; More]">
      <![CDATA[java.lang.AssertionError: expected [] but found [Lord and Taylor: Designer Clothing, Shoes, Handbags, Accessories & More]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at com.paxotech.qa.lord.HomeBasePage.veriftyTitle(HomeBasePage.java:145)
	at com.lordandTaylor.testsripts.LoginBaseFanctionality.verifyhomepagefaunctionality(LoginBaseFanctionality.java:97)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:391)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- verifyhomepagefaunctionality -->
  <testcase name="vlaidUerLogin" time="20.294" classname="com.lordandTaylor.testsripts.LoginBaseFanctionality"/>
</testsuite> <!-- FFScript -->
